IdentityCore/src/oauth2/token/MSIDPrimaryRefreshToken.m

```diff
@@ -208,7 +208,7 @@ - (NSUInteger)refreshInterval
 {
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->